components/Buttons/examples/ButtonsStoryboardAndProgrammatic.swift

```diff
@@ -18,67 +18,88 @@ import Foundation
 import MaterialComponents
 
 class ButtonsSwiftAndStoryboardController: UIViewController {
-
-  class func catalogBreadcrumbs() -> [String] {
-    return ["Buttons", "Buttons (Swift and Storyboard)"]
-  }
-
-  class func catalogStoryboardName() -> String {
-    return "ButtonsStoryboardAndProgrammatic"
-  }
-
-  class func catalogIsPrimaryDemo() -> Bool {
-    return false
-  }
-
+  
   let raisedButton = MDCRaisedButton()
   let flatButton = MDCFlatButton()
   let floatingButton = MDCFloatingButton()
-
+  
```

Comment:
Formatting: Please remove the extra white space on the empty lines here and below.